npm/ng-packs/packages/oauth/src/lib/oauth.module.ts

```diff
@@ -60,6 +61,7 @@ export class AbpOAuthModule {
         },
         OAuthModule.forRoot().providers as Provider[],
         { provide: OAuthStorage, useClass: AbpLocalStorageService },
+        { provide: AuthErrorFilterService, useExisting: OAuthErrorFilterService },
```

Comment:
why useExistsing? why not useClass?

Reply:
ProvidedIn root

![image](https://github.com/abpframework/abp/assets/49063256/e301ec47-7923-4342-80d1-6d9ddce78889)